utils.py

```diff
@@ -20,33 +35,42 @@ def parse_long_dates(date_string):
         month (str): The month as a string representing an integer between 1 and 12
         day (str): The day as a string representing an integer between 1 and 31
     """
+
+
+    date_regex = re.compile(r'([\w]*)\s+(\d{1,2})\D*(\d{4})', re.IGNORECASE)
+    date_re = date_regex.search(date_string)
+    if date_re is None:
+        return False, f"'{date_string}' is not a parseable date"
```

Comment:
Just a thought here. I'm getting output like:

    '' is not a parseable date

Which is not that helpful because I don't know what the context was that `parse_long_dates()` got that input from. Might be worth passing in the `pdf_url` defined on line 112 of `utils.py` in to `parse_long_dates()` as well, in order to provide a more helpful error message.

Reply:
This is expected behavior, since there are three links on that page that have just a single whitespace as the link text. I posted a screenshot of the page with the inspector showing the whitespace for link text on Oct 11 in the general channel of the slack. The current output should let you know which year page the bad link was on, which should be enough to find the bad link itself and determine if there's an error in the code or the page.

Reply:
I like the idea of including a bit more context though, which will help us find the source files for the non-parseable dates. Luckily we know the source of these issues, but it would be helpful in the future to include the page and the link as well so we can find it in the source code for that page